go/chat/attachments/preview_darwin.go

```diff
@@ -62,6 +64,29 @@ int ImageLength() {
 int VideoDuration() {
 	return duration;
 }
+
+void HEICToJPEG(const char* inFilename) {
```

Comment:
Should probably do better error handling here. Can you some global variable like we do for the result to make it easier to interface with Go.

Reply:
That is, if this function fails, we just send the HEIC image as a file, but not sending a blank attachment or whatever would happen.